apps/persons/migrations/0002_auto_20180518_1636.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
replay makemigrations to have only one migration 0001 (0002 and 0003 are due to modifications that you did while coding)

Reply:
Or squash them!